### ISSUE TITLE
Edit composer remains highlighted when emoji picker menu is opened.

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#14798]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Open the web app
2. Open any report having text message.
3. Click on edit message icon to edit message.
4. Click on emoji picker icon in edit message composer.

## Expected Result:
Edit composer shouldn’t be highlighted

## Actual Result:
Edit composer remains highlighted.

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.2.64-2
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 
![Screen Shot 2023-01-31 at 4 41 46 PM](https://user-images.githubusercontent.com/43996225/216619950-5f240446-8267-4b4c-a4a3-9e4f37f2668f.png)
<img width="960" alt="1" src="https://user-images.githubusercontent.com/43996225/216620015-83d03392-2682-45a2-8124-88bb306c7d93.png">

**Expensify/Expensify Issue URL:**
**Issue reported by:** @aneequeahmad 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1675165374280019

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by tienifr** [comment]
# Proposal

This is due to we're preventing the execution of the `onBlur` event of the composer if the emoji button is clicked. The following will fix it:

```diff
diff --git a/src/pages/home/report/ReportActionItemMessageEdit.js b/src/pages/home/report/ReportActionItemMessageEdit.js
index 97fffebebc..6c885f5218 100644
--- a/src/pages/home/report/ReportActionItemMessageEdit.js
+++ b/src/pages/home/report/ReportActionItemMessageEdit.js
@@ -243,7 +243,7 @@ class ReportActionItemMessageEdit extends React.Component {
                         }}
                         onBlur={(event) => {
                             // Return to prevent re-render when save/cancel button is pressed which cancels the onPress event by re-rendering
-                            if (_.contains([this.saveButtonID, this.cancelButtonID, this.emojiButtonID], lodashGet(event, 'nativeEvent.relatedTarget.id'))) {
+                            if (_.contains([this.saveButtonID, this.cancelButtonID], lodashGet(event, 'nativeEvent.relatedTarget.id'))) {
                                 return;
                             }
                             this.setState({isFocused: false});

```

**Comment by tienifr** [comment]
Working well after the fix


https://user-images.githubusercontent.com/113963320/216741978-2644c0c4-35bb-4a09-bb14-e300ab7db75f.mov

**Comment by tienifr** [comment]
Furthermore, I also found another issue on mweb (not related to the fix and the current PR, but related for the highlighting composer issue). You can check it out here
https://expensify.slack.com/archives/C049HHMV9SM/p1675481157349539

**Comment by jatinsonijs** [comment]
**Root Case**

```
if (_.contains([this.saveButtonID, this.cancelButtonID, this.emojiButtonID], lodashGet(event, 'nativeEvent.relatedTarget.id'))) {
  return;
 }
 this.setState({isFocused: false});
 toggleReportActionComposeView(true, this.props.isSmallScreenWidth);
 ```
 
 Due to above condition isFocused not set to false and Input still look like as focused. We are using this condition for different purpose (to prevent calling of toggleReportActionComposeView in specific condition).
 
 **Proposal**
 
 We should always set isFocus false on blur, because if onBlur is called that's mean Input is no more focused now, in this case user cannot type anything in it so there is no reason to show input as focused.
 
 ```diff
 diff --git a/src/pages/home/report/ReportActionItemMessageEdit.js b/src/pages/home/report/ReportActionItemMessageEdit.js
index 97fffebebc..272392b941 100644
--- a/src/pages/home/report/ReportActionItemMessageEdit.js
+++ b/src/pages/home/report/ReportActionItemMessageEdit.js
@@ -242,11 +242,12 @@ class ReportActionItemMessageEdit extends React.Component {
                             toggleReportActionComposeView(false, this.props.isSmallScreenWidth);
                         }}
                         onBlur={(event) => {
+                            this.setState({isFocused: false});
+
                             // Return to prevent re-render when save/cancel button is pressed which cancels the onPress event by re-rendering
                             if (_.contains([this.saveButtonID, this.cancelButtonID, this.emojiButtonID], lodashGet(event, 'nativeEvent.relatedTarget.id'))) {
                                 return;
                             }
-                            this.setState({isFocused: false});
                             toggleReportActionComposeView(true, this.props.isSmallScreenWidth);
                         }}
                         selection={this.state.selection}
```

**Comment by aneequeahmad** [comment]
@kavimuru Please mention my optional proposal from [slack thread](https://expensify.slack.com/archives/C049HHMV9SM/p1675530384463249?thread_ts=1675165374.280019&cid=C049HHMV9SM).

As a bug reporter my [optional solution](https://expensify.slack.com/archives/C049HHMV9SM/p1675530384463249?thread_ts=1675165374.280019&cid=C049HHMV9SM) should be evaluated first. Thanks

**Comment by aneequeahmad** [comment]
@rushatgabhane Please look at the guideline [here](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#:~:text=whilst%20you%20may%20optionally%20propose%20a%20solution%20for%20that%20job%20on%20Slack%2C%20solutions%20are%20ultimately%20reviewed%20in%20GitHub.%20The%20onus%20is%20on%20you%20to%20propose%20the%20solution%20on%20GitHub%2C%20and/or%20ensure%20the%20issue%20creator%20will%20include%20a%20link%20to%20your%20proposal.) it says `you may optionally propose a solution for that job on Slack, solutions are ultimately reviewed in GitHub. The onus is on you to propose the solution on GitHub, and/or ensure the issue creator will include a link to your proposal.` 

I mentioned the issue creator @kavimuru  in the slack thread to include a link to my proposal. It has happened for the 2nd or 3rd time that my optional solution is not included in the issue. Let me know if it makes sense or i'm missing something

**Comment by strepanier03** [comment]
@aneequeahmad - I've reviewed the guidelines and it doesn't say that a solution proposed in Slack by the bug reporter is required to be considered before other proposals. 

It does say that you can optionally propose a solution in Slack but it's your responsibility to ensure it's added to the issue created for the bug. In this case, we'd already received proposals on the GH by the time you posted on the GH about your proposal, and it was almost 24 hours after the issue was created that your solution was proposed in Slack.

I would recommend posting your proposal on the GH rather than in Slack and relying on a third party to post your solution.

**Comment by aneequeahmad** [comment]
@strepanier03 Please look at the slack thread [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1670952850266589). I'm pretty sure that the bug reporter has the priority that their proposal is evaluated first. 

> t it's your responsibility to ensure it's added to the issue created for the bug

I tagged the issue reporter(@kavimuru) in the slack thread [here](https://expensify.slack.com/archives/C049HHMV9SM/p1675285954631569?thread_ts=1675165374.280019&cid=C049HHMV9SM) on `Feb 2nd` to link my [optional solution ](https://expensify.slack.com/archives/C049HHMV9SM/p1675165716399439?thread_ts=1675165374.280019&cid=C049HHMV9SM) with the issue. 

Again, i asked the issue reporter on `Feb 4th` to  link my optional solution with Github issue when this issue was created
I can't comment on Github issue as it is locked before Help Wanted` label is added.

>  In this case, we'd already received proposals on the GH by the time you posted your proposal in Slack

I posted my proposal in slack on `Jan 31st` [here](https://expensify.slack.com/archives/C049HHMV9SM/p1675165716399439?thread_ts=1675165374.280019&cid=C049HHMV9SM).

> I would recommend posting your proposal on the GH rather than in Slack and relying on a third party to post your solution.

Yes, you're right Github is the right place to post and evaluate proposals. But how can we make sure the bug reporter has the right to post proposal on Github when their is time difference and other factors. In my case When i woke up `Help wanted` label was added and there were proposals already posted here. 

CC: @mallenexpensify

**Comment by mallenexpensify** [comment]
Apologies for the confusion, I'll try to summarize first.  For a long time (~2 years) we wanted to ensure that bugs got fixed as quickly as possible, in order to do so we wanted proposals submitted quickly and I updated CONTRIBUTING.md months(?) ago to reflect that proposals submitted in Slack threads would be posted upon GH issue creation.  

Starting a month or two ago we updated the process so that proposals need to be submitted in GH in order to be considered.  Since then CONTRIBUTING.md has been updated again, to what you posted above, which has been removed, 
> you may optionally propose a solution for that job on Slack, solutions are ultimately reviewed in GitHub. The onus is on you to propose the solution on GitHub, and/or ensure the issue creator will include a link to your proposal.

So... where we're at now 
- CONTRIBUTING.md was just updated and we're asking contributors so use a template for proposing fixes - [here](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#propose-a-solution-for-the-job)
- Contributor's who report bugs in #expensify-bugs do *not* have dibs or priority to propose a solution first.  Once the `Help Wanted` label is added, the issue is open for proposals

>  But how can we make sure the bug reporter has the right to post proposal on Github when their is time difference and other factors. In my case When i woke up Help wanted label was added and there were proposals already posted here.

There's nothing we can do about this now as we need our process to be scalable and we want to ensure that all bugs get fixed promptly.   

For this specific issue, since CONTRIBUTING.md stated at the time that proposals should be honored in bug reports, I think @aneequeahmad 's should be considered first.   @aneequeahmad, can you repost it here again using the new template?  Thx.

**Comment by aneequeahmad** [comment]
### Proposal

**Problem**

Edit composer remains highlighted which shouldn’t be highlighted.

**Root cause of problem**
In the component `ReportActionItemMessageEdit` we are setting the state isFocused false after the condition 
```
if (_.contains([this.saveButtonID, this.cancelButtonID, this.emojiButtonID], lodashGet(event, 'nativeEvent.relatedTarget.id'))) {
    return;
}
```

 we are returning here which prevents the isFocused state to set to false and this state is only used to set the composer focused color here
 https://github.com/Expensify/App/blob/3ce01b1fd44b57c5b0b16d1ed9a49f71e6691f9d/src/pages/home/report/ReportActionItemMessageEdit.js#L224

**Changes to solve the problem**

We need to set the isFocused to false at the start of onBlur before the check which is returning and preventing to set state of `isFocused` to `false`

code changes:
```diff
                         onBlur={(event) => {
+                            this.setState({isFocused: false});
                             // Return to prevent re-render when save/cancel button is pressed which cancels the onPress event by re-rendering
                             if (_.contains([this.saveButtonID, this.cancelButtonID, this.emojiButtonID], lodashGet(event, 'nativeEvent.relatedTarget.id'))) {
                                 return;
                             }
-                            this.setState({isFocused: false});
                             toggleReportActionComposeView(true, this.props.isSmallScreenWidth);
                         }}

```

**Comment by strepanier03** [comment]
Thank you @mallenexpensify for the context and @aneequeahmad for reporting. @rushatgabhane, please review the proposal from @aneequeahmad as a first choice this time. Moving forward the bug reporter's proposal will not get priority.

**Comment by aneequeahmad** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.

- Edit composer remains highlighted which shouldn’t be highlighted.

### What is the root cause of that problem?

- In the component ReportActionItemMessageEdit we are setting the state isFocused false after the condition
```
if (_.contains([this.saveButtonID, this.cancelButtonID, this.emojiButtonID], lodashGet(event, 'nativeEvent.relatedTarget.id'))) {
    return;
}
```
we are returning here which prevents the isFocused state to set to false and this state is only used to set the composer focused color here

### What changes do you think we should make in order to solve the problem?


- We need to set the isFocused to false at the start of onBlur before the check which is returning and preventing to set state of isFocused to false

```
this.setState({isFocused: false});

// Return to prevent re-render when save/cancel button is pressed which cancels the onPress event by re-rendering
 
 if (_.contains([this.saveButtonID, this.cancelButtonID, this.emojiButtonID], lodashGet(event, 'nativeEvent.relatedTarget.id'))) {
               return;
  }
```



cc: @rushatgabhane

**Comment by jatinsonijs** [comment]
**Proposal**

**Please re-state the problem that we are trying to solve in this issue.**
Edit composer Text Input remains highlighted even when it is not focused

**What is the root cause of that problem?**
We are using **isFocused** state to determine edit composer should show as highlight or not. when text input is focused we are setting this state as true.
and it should be set as false on blur input, but in on blur function we are using some conditions which is preventing to reach setState false of isFocused.

```
if (_.contains([this.saveButtonID, this.cancelButtonID, this.emojiButtonID], lodashGet(event, 'nativeEvent.relatedTarget.id'))) {
               return;
  }
  ```
 this condition we are using for different purpose there is not intention to prevent isFocused state changes. We are using this condition to prevent **toggleReportActionComposeView** execution to stop re-rendering and it is related to this bug https://github.com/Expensify/App/issues/10290

**What changes do you think we should make in order to solve the problem?**
We should always set isFocus false on blur, because if onBlur is called that's mean Input is no more focused now, in this case user cannot type anything in it so there is no reason to show input as highlighted.

to solve this shift this.setState({isFocused: false}); at top of onBlur function.

**Comment by tienifr** [comment]
I also have another proposal, originally [here](https://github.com/Expensify/App/issues/14798#issuecomment-1416595293), let me rewrite it with the new template

## Proposal

### Please re-state the problem that we are trying to solve in this issue.
After clicking emoji button on Desktop, edit composer Text Input remains highlighted while it should not

### What is the root cause of that problem?
We're preventing the execution of `this.setState({isFocused: false});` of the composer if the emoji button is clicked, which we should not, we should only avoid calling the `toggleReportActionComposeView(true, this.props.isSmallScreenWidth);` when the emoji button is clicked.

### What changes do you think we should make in order to solve the problem?


We should move the following check
```
if (this.emojiButtonID === lodashGet(event, 'nativeEvent.relatedTarget.id')) {
                                return;
                            }
```
to below the `this.setState({isFocused: false});`.

Pls note that we still need to prevent the toggleReportActionComposeView(true, this.props.isSmallScreenWidth); in case of clicking the emoji button so that we don't reintroduce [this bug](https://github.com/Expensify/App/issues/10290).

I don't think we should move `this.setState({isFocused: false});` above  `if (_.contains([this.saveButtonID, this.cancelButtonID]` because there're totally valid cases where an input still has focus after a save button is clicked (Slack on web does it, when you click Send the composer is still focused). And those cases are not related to this issue and do not cause bugs (the edit composer will be closed after clicking on those buttons) so we shouldn't touch it.

If some time in the future we don't want to close the composer when clicking Save, then the correct behavior IMO is to keep the focus on the input after clicking Save so the user can do more edits if he/she wants to.

### What alternative solutions did you explore? (Optional)
NA

**Comment by aneequeahmad** [comment]
> I don't think we should move this.setState({isFocused: false}); above if (_.contains([this.saveButtonID, this.cancelButtonID]
 
-  @tienifr Please note that `isFocused state` is only used to change the chat item box focus color on `line#227` in component `ReportActionItemMessageEdit`.  
 
 > (the edit composer will be closed after clicking on those buttons) so we shouldn't touch it.
 
 - This isn't caused when `this.setState({isFocused: false});` is moved above the check to return. Please verify at your by putting the code where i suggested.
 
 > If some time in the future we don't want to close the composer when clicking Save, then the correct behavior IMO is to keep the focus on the input after clicking Save
   
   - I don't think this would be needed if we think in terms of UX. User is done with the edit feature and now want's to close this edit comment option.

**Comment by chiragsalian** [comment]
This doesn't feel like a daily so I'm demoting it to weekly.

**Comment by strepanier03** [comment]
@chiragsalian - From what I understand with our BZ process this is supposed to stay daily to keep everyone acting with urgency. Are you feeling like this isn't daily because the issue isn't super important to fix?

**Comment by strepanier03** [comment]
@rushatgabhane - Is there any action that needs to be taken or are we just waiting for a sufficient proposal to be accepted?

**Comment by MelvinBot** [comment]
@strepanier03 @chiragsalian @rushatgabhane this issue was created 2 weeks ago. Are we close to approving a proposal? If not, what's blocking us from getting this issue assigned? Don't hesitate to create a thread in #expensify-open-source to align faster in real time. Thanks!

**Comment by chiragsalian** [comment]
Proposal LGTM as well. Feel free to create a PR @aneequeahmad.

**Comment by MelvinBot** [comment]
📣 @aneequeahmad You have been assigned to this job by @chiragsalian!
Please apply to this job in Upwork and leave a comment on the Github issue letting us know when we can expect a PR to be ready for review 🧑‍💻
Keep in mind: [Code of Conduct](https://github.com/Expensify/App/blob/main/CODE_OF_CONDUCT.md) | [Contributing](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md) 📖

**Comment by aneequeahmad** [comment]
I'll create the PR today. Thanks for mentioning @chiragsalian @rushatgabhane

**Comment by strepanier03** [comment]
@rushatgabhane - You'd still like me to undo the price jump to $4k correct?

**Comment by strepanier03** [comment]
Brilliant, handling that now and thank you for confirming.

**Comment by MelvinBot** [comment]
Upwork job price has been updated to $2000

**Comment by aneequeahmad** [comment]
@rushatgabhane apologies I raised a PR but somehow linked issue wasn't correct. Updated the PR.  Please review. Lets merge in the mean time for the bonus.

**Comment by MelvinBot** [comment]
📣 @aneequeahmad! 📣

Hey, it seems we don’t have your contributor details yet! You'll only have to do this once, and this is how we'll hire you on Upwork.
Please follow these steps:

1. Get the email address used to login to your Expensify account. If you don't already have an Expensify account, create one [here](https://new.expensify.com/). If you have multiple accounts (e.g. one for testing), please use your main account email.
2. Get the link to your Upwork profile. It's necessary because we only pay via Upwork. You can access it by logging in, and then clicking on your name. It'll look like [this](https://www.upwork.com/freelancers/~01ca8ad8ce36b9eceb). If you don't already have an account, sign up for one [here](https://www.upwork.com/nx/signup/?dest=home).
3. Copy the format below and paste it in a comment on this issue. Replace the placeholder text with your actual details.

![Screen Shot 2022-11-16 at 4 42 54 PM](https://user-images.githubusercontent.com/30609178/202326034-1a292e64-48fd-4fef-9f4f-7feaaeedfd9f.png)

Format:
```
Contributor details
Your Expensify account email: <REPLACE EMAIL HERE>
Upwork Profile Link: <REPLACE LINK HERE>
```

**Comment by aneequeahmad** [comment]
@rushatgabhane PR created. Today is the 3rd day since i was assigned the issue. Let's merge this,

cc: @chiragsalian

**Comment by aneequeahmad** [comment]
Contributor details
Your Expensify account email: aneeque65@gmail.com
Upwork Profile Link: https://www.upwork.com/freelancers/~01b86de292275cfb3b

**Comment by MelvinBot** [comment]
✅ Contributor details stored successfully. Thank you for contributing to Expensify!

**Comment by strepanier03** [comment]
Will do some additional bumps Monday if there is no movement on the PR by then.

**Comment by strepanier03** [comment]
@chiragsalian @rushatgabhane - How are we looking with the PR submitted by @aneequeahmad?

**Comment by chiragsalian** [comment]
Replied in the PR. It seems pretty small so I imagine just testing on all platforms is required and am hoping that @rushatgabhane would be able to test it out.

**Comment by strepanier03** [comment]
Head's up that I'm out of the office tomorrow so if there is an urgent need for BugZero action on this GH, please reach out in [#expensify-open-source](https://expensify.slack.com/archives/C01GTK53T8Q) to ask for help.

I'll be back Monday and will check in then.

**Comment by strepanier03** [comment]
I see the PR is merged for I'll follow along for the deploy.

**Comment by MelvinBot** [comment]
`Reviewing` label has been removed, please complete the "BugZero Checklist".

**Comment by MelvinBot** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.2.80-2 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/15482


If no regressions arise, payment will be issued on 2023-03-16. :confetti_ball:

After the hold period is over and BZ checklist items are completed, please complete any of the applicable payments for this issue, and check them off once done.
- [ ] External issue reporter
- [ ] Contributor that fixed the issue
- [ ] Contributor+ that helped on the issue and/or PR

As a reminder, here are the bonuses/penalties that should be applied for any External issue:
- Merged PR within 3 business days of assignment - 50% **bonus**
- Merged PR more than 9 business days after assignment - 50% **penalty**

**Comment by MelvinBot** [comment]
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@rushatgabhane / @chiragsalian] The PR that introduced the bug has been identified. Link to the PR:
- [ ] [@rushatgabhane / @chiragsalian] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
- [ ] [@rushatgabhane / @chiragsalian] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@strepanier03] Determine if we should create a regression test for the bug.
- [ ] [@rushatgabhane] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to the [appropriate location](https://github.com/Expensify/App/blob/main/contributingGuides/TestRail_Navigation_C%2B.md) to ensure the same bug will not reach production again.
- [ ] [@strepanier03] Review the proposed regression test steps and location.
- [ ] [@strepanier03] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by MelvinBot** [comment]
Looks like something related to `react-navigation` may have been mentioned in this issue discussion.

As a reminder, please make sure that all proposals are not workarounds and that any and all attempt to fix the issue holistically have been made before proceeding with a solution. Proposals to change our `DeprecatedCustomActions.js` files should not be accepted.

Feel free to drop a note in [#expensify-open-source](https://expensify.slack.com/archives/C01GTK53T8Q) with any questions.

**Comment by aneequeahmad** [comment]
@strepanier03 it says the offer is not available anymore.

**Comment by strepanier03** [comment]
@aneequeahmad Oh how odd, I've never run into that before. I'll resend and if it does it again I'll just make a new job.

**Comment by strepanier03** [comment]
I'm not able to send you another offer for that job so I need to make a new one but Upwork's job creation is down again.

<img width="559" alt="2023-03-15_19-38-47" src="https://user-images.githubusercontent.com/10925636/225496140-3d02fcc2-2eab-438a-8ad6-e4394add0b68.png">

I'll try again in the morning and let you know here when I send the offer.

**Comment by aneequeahmad** [comment]
@strepanier03 Oh yes, that's unusual. I'll wait for the new offer. Thanks

**Comment by aneequeahmad** [comment]
@strepanier03 any update here ?

**Comment by strepanier03** [comment]
@aneequeahmad I am still getting the error but I had some team members trying to help me. 

You will see an offer without a job connected that you can ignore. My team member also created an actual job for this and I just hired you for it.

The job is https://www.upwork.com/jobs/~01a58b143d5995e2ff.

The contract you can ignore is from Kadie Alexander. They are going to close it so you might never see it but just in case you do you can ignore it.

**Comment by strepanier03** [comment]
@rushatgabhane - Once the BugZero checklist is complete I'll be able to handle your Upwork pay out.

**Comment by aneequeahmad** [comment]
@strepanier03 accepted the offer. Thank you for mentioning

**Comment by strepanier03** [comment]
@aneequeahmad you should see a contract offer for the reporting bonus, can you take a look?

**Comment by strepanier03** [comment]
Alright, just paid that out @aneequeahmad - you should be all set. 

@rushatgabhane I'll check back in Monday to see if the checklist is finished.

**Comment by strepanier03** [comment]
Just waiting on the BZ checklist to be completed @rushatgabhane @chiragsalian, then we can wrap this one up.

**Comment by strepanier03** [comment]
[Bumped](https://expensify.slack.com/archives/C02NK2DQWUX/p1679614997898319).

**Comment by strepanier03** [comment]
@rushatgabhane - I'll check back in tomorrow after you've had time to review @parasharrajat comment [here](https://github.com/Expensify/App/issues/14798#issuecomment-1485983253).

**Comment by rushatgabhane** [comment]
Yeah I was wrong. Thanks for the help @parasharrajat 
I'll get down to the cause today

**Comment by strepanier03** [comment]
Thank you Rushat 🙌 , if you want to @ mention me when you do I can make the update in the BZ checklist and finish paying this out to you.

**Comment by MelvinBot** [comment]
@strepanier03, @chiragsalian, @rushatgabhane, @aneequeahmad Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by rushatgabhane** [comment]
@strepanier03 im not able to find the commit that caused this bug

**Comment by strepanier03** [comment]
I hid those comments from the incorrect PR and paid you out @rushatgabhane. 

Thank you all for the work you did here and for helping each other out with the final steps. Closing now!
